tests/storage-volumes-vm

```diff
@@ -191,6 +191,74 @@ do
 	lxc storage volume detach "${poolName}" vol3 v1
 	lxc storage volume detach "${poolName}" vol6 v1 || true  # optional ISO
 
+	# attach VM root volumes
+	if hasNeededAPIExtension instance_root_volume_attachment; then
+		lxc init --empty --vm v2 --storage "${poolName}" --device root,size=8KiB
+		lxc init --empty --vm v3 --storage "${poolName}" --device root,size=8KiB
```

Comment:
It's good to test with such small volumes. I'm pretty sure LVM will round them up but that should happen all transparently, hopefully!

Dunno how it would fare with PowerFlex's minimum of 8GiB though. I guess we'll see when we run it there :)

Reply:
powerflex doesnt round up to 8gib afaik @roosterfish ?

Reply:
Pushed a special case for powerflex.